.github/workflows/build_and_test.yml

```
@@ -174,6 +174,26 @@ jobs:
           submodules: true
           fetch-depth: 1
 
+      - name: Check Postgres submodules revision
+        run: |
+          # This is a temporary solution to ensure that the Postgres submodules revision is correct (i.e. the updated intentionally).
+          # Eventually it will be replaced by a regression test https://github.com/neondatabase/neon/pull/4603
+
+          FAILED=false
+          for postgres in echo "postgres-v14 postgres-v15"; do
```

```suggestion
          for postgres in postgres-v14 postgres-v15; do
```

Echo is wrong here, I think it could be bare words, lemme test.